docs/user/guides/troubleshooting/index.rst

```diff
@@ -0,0 +1,14 @@
+How to troubleshoot specific issues
+-----------------------------------
```

Comment:
```suggestion
Troubleshooting Guides
----------------------
```

The original title didn't read like an index listing, but instead like a how to article. I think we should be pretty explicit in language around index or listing pages. 

I could even be convinced on something like `Troubleshooting Guide Listing` or similarly explicit.